app/index.html

```diff
@@ -14,6 +14,14 @@
                 </a>
             </div>
 
+            <div id="top-search">
+                <img id="top-search-icon" src="img/search.png" />
+            </div>
+            
+            <div id="top-search-box">
+                <input type="text" id="top-search-input" placeholder="Search Google..."></input>
+            </div>
```

Comment:
Dont rename the div, its a part of the search box so put it inside the top-search div :smile: